react-challenge.html

```diff
@@ -42,17 +42,19 @@
 
         render(){
           return(
-          <div>
```

Comment:
awesome! remember, you can also use `React.Fragment` to avoid adding extra unnecessary nodes to your HTML :)